pthread/pthread.c

```diff
@@ -101,27 +142,31 @@ int pthread_create(pthread_t *thread, const pthread_attr_t *attr,
 		return -EAGAIN;
 	}
 
+	pthread_ctx_get(ctx);
```

Comment:
In function `pthread_ctx_get(ctx)` you increment uninitialized `ctx->refcount` variable.
Why not:
```suggestion
	ctx->refcount = 1;
```

Reply:
Of course, thanks.